desktop-widgets/divelistview.cpp

```diff
@@ -596,7 +596,7 @@ void DiveListView::selectionChanged(const QItemSelection &selected, const QItemS
 	std::vector<dive *> selection = getDiveSelection();
 	updateSelection(selection, addToSelection, removeFromSelection);
 	dive *newCurrent = selection.empty() ? nullptr : selection.front();
-	::setSelectionCore(selection, newCurrent, -1);
+	setSelectionCore(selection, newCurrent, -1);
```

Comment:
unless I'm missing something, maybe this could be moved into the previous commit (so we don't use that notation in the first place...)

Reply:
fixed.